plat/xilinx/zynqmp/include/plat_macros.S

```diff
@@ -41,7 +41,7 @@
 	 * Clobbers: x0 - x10, x16, sp
 	 * ---------------------------------------------
 	 */
-	.macro plat_print_gic_regs
+	.macro plat_crash_print_regs
 	mov_imm	x17, BASE_GICC_BASE
 	mov_imm	x16, BASE_GICD_BASE
 	arm_print_gic_regs
```

Comment:
Should also call `print_cci_regs`. (BTW I'm not sure what the below `mov` instruction is for but that's not your problem).